fhservices.cfg.j2

```diff
@@ -184,6 +194,14 @@ define service {
 }
 {% endif %}
 
+define service {
+       service_description jenkins:ping
+       check_command check_jenkins!80!/login
```

Comment:
80 is the only port that is listening when tested.  If it should be 443, something else may be wrong as it returns a no route to host when trying to get to jenkins:443